app.js

```diff
@@ -4,11 +4,12 @@ var ari = require('ari-client');
 var util = require('util');
 var sla = require('./lib/sla.js');
 var Q = require('q');
+var dal = require('./lib/dal.js');
 
 var connect = Q.denodeify(ari.connect);
 var confFile;
 if (confFile = process.argv[2]) {
-  connect('http://127.0.0.1:8088', 'user', 'pass')
+  connect('http://localhost:8088', 'user', 'pass')
```

Comment:
I agree with Matt's comment. When you are building software, remember to keep the changes small and targeted. It is a lot easier to find the source of a regression if you only change one variable at a time.